inst/stan/wwinference.stan

```diff
@@ -6,6 +6,14 @@ functions {
 #include functions/infections.stan
 #include functions/observation_model.stan
 #include functions/utils.stan
+#include functions/exponential_decay_corr_func.stan
+#include functions/independence_corr_func.stan
+#include functions/construct_spatial_rt.stan
+#include functions/spatial_deviation_noise_matrix_rng.stan
+#include functions/spatial_rt_process_rng.stan
+#include functions/construct_aux_rt.stan
+#include functions/state_deviation_noise_vec_aux_rng.stan
+#include functions/aux_site_process_rng.stan
```

Comment:
If these aren't used in the actual model yet, I think best not to `#include` them here. I'm guessing the goal is to expose them to R, I would suggest for now writing a general `functions.stan` file that includes them all and using `cmdstanr`'s `$expose_functions()` functionality on that file.